CHANGELOG.md

```diff
@@ -6,19 +6,90 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/).
 
 ## Upcoming release
 
+nothing yet
+
+## [0.10.0] - 2019-09-03
+
 ### Highlights
 
 * The minimum supported Bazel version is now 0.27.
+
   `rules_haskell` supports Bazel up to 0.28.
   0.27 is a LTS release, which means upstream guarantees all new
   releases are backwards-compatible to it for 3 months. See the [Bazel
   Stability](https://blog.bazel.build/2019/06/06/Bazel-Semantic-Versioning.html)
   blog post for more information.
+
 * The repository name has changed, to follow the
   new [Bazel rule guidelines][bazel-rule-guidelines]. It was
   previously called `@io_tweag_rules_haskell`. It is now called
   `@rules_haskell`. You should adapt your `WORKSPACE` file to match
   this, as well as your `.bazelrc.local` file, if any.
+
+* `haskell_cabal_library`/`haskell_cabal_binary`: Both use `cabal` to
+  build Haskell package dependencies.
+
+* `stack_snapshot`: Uses stack’s dependency resolving algorithm to
+  generate `haskell_cabal_library` targets automatically.
+  Requires `stack` > 2.1, which it will download if cannot be found in
+  `PATH`.
+
+* It is now possible to statically link Haskell libraries in CC
```

Comment:
Just to clarify for myself, this is referring to 0ff1690525a676725401081bc646189378071c40?

Reply:
Yes, did I interpret this commit message incorrectly?

Reply:
Yes, looks good to me.